web/app/controllers/Application.java

```diff
@@ -93,6 +94,38 @@ public static Result index(String path) {
         return serveAsset("");
     }
 
+    /**
+     * Creates a wrapping ObjectNode containing config information
+     *
+     * @return Http Result instance with app configuration attributes
+     */
+    public static Result appConfig() {
+        ObjectNode response = Json.newObject();
+        Boolean isInternal = Play.application().configuration().getBoolean(LINKEDIN_INTERNAL_KEY, false);
```

Comment:
How about defining this as a private static final at the top of this class since it's not expected to change and might be used at multiple places?